Makefile

```diff
@@ -0,0 +1,37 @@
+ENVIRONMENT ?= betacloud
+
+VERSION_CEPH ?= pacific
+VERSION_MANAGER ?= latest
+VERSION_OPENSTACK ?= yoga
+
+clean:
+	cd terraform && make ENVIRONMENT=$(ENVIRONMENT) clean
+
+create:
+	cd terraform && make ENVIRONMENT=$(ENVIRONMENT) VERSION_CEPH=$(VERSION_CEPH) VERSION_MANAGER=$(VERSION_MANAGER) VERSION_OPENSTACK=$(VERSION_OPENSTACK) create
+
+login:
+	cd terraform && make ENVIRONMENT=$(ENVIRONMENT) login
+
+deploy:
+	ansible-playbook playbooks/deploy.yml \
+	  -i ansible/localhost_inventory.yaml \
+	  -e ansible_galaxy=ansible-galaxy \
+	  -e ansible_playbook=ansible-playbook \
+	  -e basepath="$(PWD)" \
+	  -e cloud_env=$(ENVIRONMENT) \
+	  -e repo_path="$(PWD)/.src/github.com"
+	  -e manual_deploy=true \
+	  -e version_ceph=$(VERSION_CEPH) \
+	  -e version_manager=$(VERSION_MANAGER) \
+	  -e version_openstack=$(VERSION_OPENSTACK)
+	cd terraform && make ENVIRONMENT=$(ENVIRONMENT) deploy
+	cd terraform && make ENVIRONMENT=$(ENVIRONMENT) check
+
+prepare:
+	mkdir -p .src/github.com/osism
+	if [[ ! -e .src/github.com/osism/testbed ]]; then git clone https://github.com/osism/testbed .src/github.com/osism/testbed; fi
+	if [[ ! -e .src/github.com/osism/ansible-collection-commons ]]; then git clone https://github.com/osism/ansible-collection-commons .src/github.com/osism/ansible-collection-commons; fi
+	if [[ ! -e .src/github.com/osism/ansible-collection-services ]]; then git clone https://github.com/osism/ansible-collection-services .src/github.com/osism/ansible-collection-services; fi
```

Comment:
These repos are only needed in order to be able to test changes to these repos in a deployment. Instead of cloning them here, we could just stop using them if they don't exist.

Reply:
Can be fixed later. For the time being, the behaviour as it was is adopted 1:1.